packages/ui/core-components/src/lib/atoms/inputs/date-input/DateInput.svelte

```diff
@@ -0,0 +1,128 @@
+<script context="module">
+	export const evidenceInclude = true;
+</script>
+
+<script>
+	import DateInput from './_DateInput.svelte';
+	import { Query } from '@evidence-dev/sdk/usql';
+	import { getQueryFunction } from '@evidence-dev/component-utilities/buildQuery';
+	import { getLocalTimeZone } from '@internationalized/date';
+	import HiddenInPrint from '../shared/HiddenInPrint.svelte';
+	import { page } from '$app/stores';
+	import QueryLoad from '$lib/atoms/query-load/QueryLoad.svelte';
+	import { Skeleton } from '$lib/atoms/skeletons/index.js';
+	import { getInputContext } from '@evidence-dev/sdk/utils/svelte';
+	import { toBoolean } from '../../../utils.js';
+	import { dateToYYYYMMDD, formatDateString } from './helpers.js';
+
+	const inputs = getInputContext();
+
+	/** @type {string} */
+	export let name;
+	/** @type {string | undefined} */
+	export let title;
+	/** @type {boolean} */
+	export let hideDuringPrint = true;
+
+	/** @type {string | Date | undefined} */
+	export let start;
+	/** @type {string | Date | undefined} */
+	export let end;
+
+	/** @type {Query | string | undefined} */
+	export let data;
+	/** @type {string | undefined} */
+	export let dates;
+	/** @type {[]string | undefined} */
+	export let presetRanges;
+	/** @type {string | undefined} */
+	export let defaultValue;
+	/** @type {boolean} */
+	export let range = false;
+
+	$: range = toBoolean(range);
+
+	const exec = getQueryFunction();
```

Comment:
@ItsMeBrianD is there an alternative we should prefer to direct `Query.create`/`getQueryFunction` here